Source/OxyPlot/Utilities/ArrayExtensions.cs

```diff
@@ -9,11 +9,36 @@
 
 namespace OxyPlot
 {
+    using System;
+    using System.Collections.Generic;
+
     /// <summary>
     /// Provides useful extension methods for arrays.
     /// </summary>
     public static class ArrayExtensions
     {
+        /// <summary>
+        /// Finds the maximum value in the sequence, or returns 0 if the sequence is empty.
+        /// </summary>
+        /// <param name="sequence">The sequence.</param>
+        /// <returns>The maximum value of the sequence, or 0 if the sequency is empty.</returns>
+        public static double MaxOrZero(this IEnumerable<double> sequence)
```

Comment:
Seems a tad specific. If it took a default value as a parameter that would be nice (the deafult value is a concern for the calling domain), but I'm not sure it's worth the effortof coming up with a different name.

Reply:
Maybe we could just change it to `MaxOrDefault<T>(...)` and return `default(T)` if the sequence is empty?

Reply:
The issue is that in either case the default value for e.g. doubles is meaningful (i.e. it's not something like `null` which explodes if you use it). I'd suggest that the default should be provided by the calling code, because only it can know if there is a meaningful default (if there isn't, it should be using `Max` and throwing or something like `TryGetMax`). I don't think it's a huge issue, but providing a default default (heh) of zero makes the method limited and easy to misuse with `structs`.

I don't think this is a big issue, so feel free to ignore it, but it feels odd for a `public` method.

Reply:
Replaced this by `MaxOrDefault` with a user-defined default value.
Also added a corresponding `MinOrDefault` because it would feel weird otherwise.